README.md

```diff
@@ -37,7 +37,19 @@ Read more about jsDelivr versioning on their [website](http://www.jsdelivr.com/)
 
 ## Configuration
 
-Any date adapter options in the chart configuration will be passed through to Luxon's factory functions. See the [Luxon API docs](https://moment.github.io/luxon/api-docs/index.html#datetimefrommillis) for the supported options, such as `zone` and `locale`, that these functions accept.
+Any date adapter options in the chart configuration will be passed through to Luxon's factory functions.
+
+The following table describes the supported options, that these functions accept, which can be set to chart [time or time series scale](https://www.chartjs.org/docs/latest/axes/cartesian/time.html).
```

Comment:
The first comma is not necessary. I'm not quite sure what the second half of the sentence is trying to communicate
```suggestion
The following table describes the supported options that these functions accept.
```